src/bors/merge_queue.rs

```diff
@@ -702,4 +610,128 @@ mod tests {
             &["main-sha1", "merge-0-pr-1", "merge-1-pr-3", "merge-2-pr-2"],
         );
     }
+
+    #[sqlx::test]
+    async fn auto_build_push_conflict_comment(pool: sqlx::PgPool) {
+        run_merge_queue_test(pool, async |tester: &mut BorsTester| {
+               start_auto_build(tester).await?;
```

Comment:
This is using internal API in a test, which I'd like to avoid, because it then tests things that don't happen in the real-world. If you often need to test a PR having a successful auto build, could you please create a helper test function that will approve the PR, run the merge queue and let it start the auto build instead?